azurerm/resource_arm_action_group_test.go

```diff
@@ -0,0 +1,296 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAzureRMActionGroup_basic(t *testing.T) {
+	resourceName := "azurerm_action_group.test"
+	ri := acctest.RandInt()
+	config := testAccAzureRMActionGroup_basic(ri, testLocation())
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMActionGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMActionGroupExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "enabled", "true"),
+					resource.TestCheckResourceAttr(resourceName, "email_receiver.#", "2"),
+					resource.TestCheckResourceAttr(resourceName, "email_receiver.0.email_address", "admin@contoso.com"),
+					resource.TestCheckResourceAttr(resourceName, "email_receiver.1.email_address", "devops@contoso.com"),
+					resource.TestCheckResourceAttr(resourceName, "sms_receiver.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "sms_receiver.0.country_code", "1"),
+					resource.TestCheckResourceAttr(resourceName, "sms_receiver.0.phone_number", "1231231234"),
+					resource.TestCheckResourceAttr(resourceName, "webhook_receiver.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "webhook_receiver.0.service_uri", "http://example.com/alert"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccAzureRMActionGroup_importBasic(t *testing.T) {
```

Comment:
Can we delete the import function and add the import test as an additional test step in the existing resource test? Sorta like this (do this for all import tests in the PR)...

```
Steps: []resource.TestStep{ 
	{ 
		Config: config, 
		Check: resource.ComposeTestCheckFunc( 
			testCheckAzureRMActionGroupExists(resourceName), 
			resource.TestCheckResourceAttr(resourceName, "enabled", "true"), 
			resource.TestCheckResourceAttr(resourceName, "email_receiver.#", "2"), 
			resource.TestCheckResourceAttr(resourceName, "email_receiver.0.email_address", "admin@contoso.com"), 
			resource.TestCheckResourceAttr(resourceName, "email_receiver.1.email_address", "devops@contoso.com"), 
			resource.TestCheckResourceAttr(resourceName, "sms_receiver.#", "1"), 
			resource.TestCheckResourceAttr(resourceName, "sms_receiver.0.country_code", "1"), 
			resource.TestCheckResourceAttr(resourceName, "sms_receiver.0.phone_number", "1231231234"), 
			resource.TestCheckResourceAttr(resourceName, "webhook_receiver.#", "1"), 
			resource.TestCheckResourceAttr(resourceName, "webhook_receiver.0.service_uri", "http://example.com/alert"), 
		), 
	},
	{ 
		ResourceName:      resourceName, 
		ImportState:       true, 
		ImportStateVerify: true, 
	}, 

```

Reply:
I've changed the test cases.